setup1-contributor/src/commands/contribute.rs

```diff
@@ -808,6 +808,8 @@ async fn start_contributor(opts: &ContributeOptions, public_settings: &PublicSet
 
     // Run the contributor.
     let contribution = match curve_kind {
+        // NOTE: the run_and_catch_errros is not implemented for Bls12_381
+        CurveKind::Bls12_381 => contribute.run_and_catch_errors::<Bls12_377>().await,
```

Comment:
Maybe should be liek this?

```suggestion
        CurveKind::Bls12_381 => contribute.run_and_catch_errors::<Bls12_381>().await,
```